src/components/Stepper/index.js

```diff
@@ -1,23 +1,24 @@
-import React, { useState, useMemo } from 'react';
-import PropTypes from 'prop-types';
-import { StepperProvider } from './StepperContext';
-import StepList from './js/StepList';
-import StepperButtons from './js/StepperButtons';
-import './stepper.css';
+import React, { useState, useMemo } from "react";
```

Comment:
I see you're changing the js strings to use double quotes instead of single quotes. I think the rule we generally use is to use single quotes. The linter also complains about this.